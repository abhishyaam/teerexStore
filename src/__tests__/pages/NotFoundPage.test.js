import { render, screen } from '@testing-library/react';
import NotFoundPage from '../../pages/NotFoundPage';

describe('<NotFoundPage />', () => {
  it('should render', () => {
    const { container, queryByText } = render(<NotFoundPage />);
    expect(queryByText('404 Page Not Found')).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = render(<NotFoundPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
