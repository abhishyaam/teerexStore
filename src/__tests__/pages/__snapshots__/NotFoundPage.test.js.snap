// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match snapshot 1`] = `
<div>
  404 Page Not Found
</div>
`;
